src/Bicep.Core.Samples/Files/Metadata_CRLF/main.symbolicnames.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "languageVersion": "1.9-experimental",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "EXPERIMENTAL_WARNING": "Symbolic name support in ARM is experimental, and should be enabled for testing purposes only. Do not enable this setting for any production usage, or you may be unexpectedly broken at any time!",
```

Comment:
I think we should name this `_EXPERIMENTAL_WARNING` to keep the distinction between system-generated and user-added metadata clear.